capa_plugin_ida.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2020 FireEye, Inc. All Rights Reserved.
```

Comment:
I'd expect to find this file in `capa/ida/plugin`?!

Reply:
The file can be moved to `capa/ida/plugin`, no problem.